google-http-client/src/main/java/com/google/api/client/util/Base64.java

```diff
@@ -26,6 +26,10 @@
  */
 @Deprecated
 public class Base64 {
+  // Special decoders that discards the new line character so that the behavior matches what
+  // we had with Apache Commmons Codec's decodeBase64.
```

Comment:
I think it would be worth commenting on what the 64 means. (It doesn't mean very much, since it's only used for encoding and we don't do that.)

Reply:
Added explanation of the 2nd parameter of the withSeparator method.